Case task sidebar enhancements

adrianwhite opened the pull request:
@cornae you may want to review these changes.
I added an extra class to the div.application so we can distinguish between a workspace type and page type. I updated the css accordingly: 04c03bd and f672b6b

Unfortunately I cannot sort #58 as this is an issue in pat-stacks

This pull fixes #3

Commit:
Whitespace

Commit:
add class for workspace type to application wrapper
change page-type to actually be type of context (when navigated to directly)

Commit:
whitespace

Commit:
change all css instance of type-case to workspace-type-case

Commit:
When in a workspace of type case the task list should be the same throughout

Commit:
Whitespace

Commit:
Tidy up workspace tabs and 'current' to events.

cornae commented:
So that's the same ' type-case' and ' type-…' that I already added there, but now it appears on all pages within a workspace context and not only the workspace landing page, right?

adrianwhite commented:
I changed it as I could forsee where we would want to know the type of the context we are on not just the type of the workspace we are in.

Commit:
Links to tasks in sidebar and on timeline now include #fragment to open tasks in sidebar.

Ideally this should be done in the template for a task if pat-stacks were configurable rather than just using fragments on initialisation.

Commit:
Merge branch 'master' into case-task-sidebar-enhancements

Conflicts:
	prototype/_includes/workspace-sidebar.html
